### DISCUSSION TITLE
yarn workspaces foreach: Preserve colors in output printed from my scripts

### DISCUSSION THREAD

**Discussion by kubijo** [yarnpkg/berry#2551] [Ideas]
Hi.

I have some CLI scripts that print coloured messages,  however when they run in `workspaces foreach` (regardless of the `-p` flag) only the yarn prefix is coloured and my messages are not… I suppose that this is intended behaviour, but I believe I should be able to at least say if I want to keep the colours intact.

What is your take on this?

**Comment by arcanis** [comment]
Duplicate with https://github.com/yarnpkg/berry/issues/2548

**Reply by kubijo** [reply]
I see, thanks for the pointer ... maybe it's worth mentioning this in the docs (`FORCE_COLOR=1`)
